lib/suite-builder.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
```

Comment:
You can simply create static hash in `lib/tests-api.js`, place where `SuiteBuilder` located now and store your functions there, without extracting anything.